### ISSUE TITLE
Button styling for search and replace

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor5#9881]

**Comment by mateuszzagorski** [comment]
Button styling for find and replace has been implemented as well, following the designs ([#9719](https://github.com/ckeditor/ckeditor5/issues/9719)) from Magda. The latest code is here: [fa2c3b1](https://github.com/ckeditor/ckeditor5/commit/fa2c3b11cce1688f7a85f9fecbc2f87923607cae) on `i/9880` branch

I've consulted with Magda regarding the color palette and as per her latest comment we'll be using default colors for text/buttons: [#9719 (comment)](https://github.com/ckeditor/ckeditor5/issues/9719#issuecomment-861438305)

**Comment by mlewand** [comment]
```
$ git merge --no-ff i/9802
Auto-merging packages/ckeditor5-find-and-replace/src/ui/findandreplaceformview.js
CONFLICT (content): Merge conflict in packages/ckeditor5-find-and-replace/src/ui/findandreplaceformview.js
Auto-merging packages/ckeditor5-find-and-replace/src/findandreplaceui.js
Automatic merge failed; fix conflicts and then commit the result.
```

I wanted to merge the latest i/9802 branch to i/9880 but there's a conflict. Can I ask you to resolve it?

**Comment by mateuszzagorski** [comment]
Resolved conflicts here: d86c421. Branch i/9802 has been merged to i/9880.

**Comment by mateuszzagorski** [comment]
Small update to the margin of the find buttons: d1e9e12 as discussed.

**Comment by mlewand** [comment]
The styling looks fine. I have some remarks:

*   The icon should be named `findreplace.svg` instead of `find-replace.svg`.
*   The HTML class name `ck-isDisabled` is incorrectly used in this case. You're not checking whether form gets disable (in which case it would get inactive/inoperable) but instead you're checking whether someone is already (in the process of) searching. So a more appropriate name would be `ck-is-searching` (note also the naming convention).

But this can be fixed in a followup task.

**Comment by mlewand** [comment]
Merged branch `i/9880` to base `i/9802` in  c061fff. Removed the `i/9880` branch.
